src/main/deploy/constants/ScoringSetpoints.json

```diff
@@ -0,0 +1,68 @@
+{
+  "idle": {
+    "elevatorHeight": {
+      "value": 1.5,
+      "unit": "Inch"
+    }
+  },
+  "l1": {
```

Comment:
nit: can we capitalize the L in this and the other ones

at first glance I thought this said 11, 12, 13, 14 which made no sense